### PR TITLE
feat(scorecard): track Integration Service as cross-cutting (tags integration-service|ipe)

### PR DESCRIPTION
**Pull request by uipreliga**
## What

`/generate-confluence-scorecard` now treats **Integration Service** as a tag-selected, cross-cutting product — aggregated across **every** skill that maintains IS tests, not just the `uipath-platform/integration-service` directory.

A task counts toward Integration Service iff it carries the tag **`integration-service` OR `ipe`**:

| Tag | Where it lives |
|---|---|
| `integration-service` | `uipath-platform/integration-service`, `uipath-agents/coded/is_*`, `uipath-troubleshoot`, `uipath-maestro-bpmn`, `uipath-rpa` |
| `ipe` | `uipath-maestro-flow` connector tasks |

## Why

The IS row previously read only the `uipath-platform/integration-service` directory — a strict subset that silently dropped the IS tasks living in agents, maestro-flow, troubleshoot, and bpmn. This is the same strict-subset trap the command already documents for ECS/Context Grounding (which once yielded `ECS 3/3` instead of the true `13/14`).

## How

Reuses the existing cross-cutting machinery (the ECS path), generalized so one product row can match **multiple tag aliases**:

- **Phase 1 script:** `CROSS_CUTTING` now takes a list of tag aliases per row; seeded `(['integration-service','ipe'], 'Integration Service')` alongside ECS. Prints a per-row **host-skill split** so the cross-skill spread is visible.
- **Phase 2/3:** IS pass/fail + tiers + eval come from the tag block (all variants, all skills), **never** the platform `psub` bucket — documented as the strict-subset trap.
- **Coverage:** no `cross_cutting` coverage.json entry for IS exists yet → use the `uipath-platform` aggregate as a **labeled floor**, footnoted, with a pointer to register IS cross-cutting in `/test-coverage`.
- Extended footnote ⁷, the edge-case table, and anti-patterns to name IS.

## Validation

Verified against the nightly run `2026-06-22_04-03-53` (the exact dashboard run `2026-06-23_04-05-07` wasn't local, but logic is identical):

- Per-tag selection matches the dashboard exactly: `integration-service` → 12, `ipe` → 22.
- IS row = the **union** (each task counted once): overlap is **0** (the tag sets are structurally disjoint), so **12 + 22 = 34**.
- **Counting all variants is essential:** tagged tasks split across the `default` and `None` variants (e.g. `ipe` = 12 default + 10 None = 22). Single-variant scoring (as the per-skill tables use) would undercount IS to ~12.
- The extracted Phase 1 script compiles clean and produces the correct union, tiers, and host-skill split on a synthetic run.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: .claude/commands/generate-confluence-scorecard.md [hunk 1]
```diff
@@ -387,7 +395,8 @@ Exclude the generic `uip … --output json` sentinel pattern (unattributable). N
...
+- **Don't source a cross-cutting row's pass/fail from a same-named directory bucket.** ECS / Context Grounding (tag `context-grounding`) and Integration Service (tags `integration-service`/`ipe`) are TAG-selected across host skills and all variants — use the Phase 1 CROSS-CUTTING TAGS block. The `uipath-platform/context-grounding` directory/`psub` bucket is a strict subset and produced a wrong 3/3 instead of 13/14; `uipath-platform/integration-service` (~8 tasks) is the same trap for IS — it omits the IS tasks in agents/flow/troubleshoot/bpmn. Same trap for any future cross-cutting capability that also happens to have a like-named folder.
+- **Don't reduce Integration Service to the platform skill.** IS spans `uipath-platform`, `uipath-agents`, `uipath-maestro-flow`, `uipath-troubleshoot`, `uipath-maestro-bpmn` (and `uipath-rpa` when present). A task belongs to IS iff it carries the tag `integration-service` OR `ipe` — count across ALL those skills, never just the platform dir.
```
Comment (baishalighosh) [inline .claude/commands/generate-confluence-scorecard.md]:
```suggestion
- **Don't reduce Integration Service to the platform skill.** IS spans `uipath-platform`, `uipath-maestro-flow`, `uipath-troubleshoot`, `uipath-maestro-bpmn` (and `uipath-rpa` when present). A task belongs to IS iff it carries the tag `integration-service` OR `ipe` — count across ALL those skills, never just the platform dir.
```
We don't maintain the agent skill, so please keep that out of skill purview for now

**Review by baishalighosh — APPROVED** [review]
